pymkv/Timestamp.py

```diff
@@ -371,18 +396,30 @@ def splitting_timestamp(self, timestamp: str) -> None:
 
         Example
         -------
-        >>> obj.splitting_timestamp("12:34:56.789012345")
+        >>> obj = Timestamp("12:34:56.789012345")
```

Comment:
If you're gonna use >>, might as well use >> everywhere.

Reply:
I enabled doctests as part of this change.  

If you use three >>> it gets interpreted as a [Pytest](https://docs.python.org/3/library/doctest.html)

There is a mix between some that are actual valid and others that are meant to be more instructive.  In this particular case I saw it as just being an example of the api.  That's what I removed one of the > so it doesn't get executed.

Reply:
[python-doctest-skip-entire-block](https://stackoverflow.com/questions/1809037/python-doctest-skip-entire-block) is quite ugly as well.